### PR TITLE
LPS-71982 Make dateField readOnly

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test

**Comment by pedroqueiroz94** [comment]
ci:retest

File: dynamic-data-mapping-type-date/src/main/resources/META-INF/resources/date.soy [hunk 1]
```diff
@@ -22,29 +22,62 @@
 	{@param? required: bool}
 	{@param? tip: string}
 
-	{let $displayValue: $formattedValue ? $formattedValue : $predefinedValue /}
```
Comment (pedroqueiroz94) [inline dynamic-data-mapping-type-date/src/main/resources/META-INF/resources/date.soy]: Hey @alinedoleron , please, keep this line here to maintain consistency with the other fields' soy files.

Comment (alinedoleron) [reply dynamic-data-mapping-type-date/src/main/resources/META-INF/resources/date.soy]: Hi, @pedroqueiroz94. @matuzalemsteles told that Infrastructure team is adopting place the variable next to where it is being used. So he requested me to do this way.
https://hosting-liferayfrontendguidelines.wedeploy.io/soy/

Comment (pedroqueiroz94) [reply dynamic-data-mapping-type-date/src/main/resources/META-INF/resources/date.soy]: oh, okay!

File: dynamic-data-mapping-type-date/src/main/resources/META-INF/resources/date_field.js [hunk 2]
```diff
@@ -96,13 +99,7 @@ AUI.add(
...
 
-						if (instance.get('readOnly')) {
-							triggerNode = container.one('.trigger-readonly');
-						}
-						else {
-							triggerNode = container.one('.trigger');
-						}
-
+						triggerNode = container.one('.form-control');
 						return triggerNode;
```
Comment (pedroqueiroz94) [inline dynamic-data-mapping-type-date/src/main/resources/META-INF/resources/date_field.js]: Should we have a blank line between these two statements? @alinedoleron

**Comment by diegonvs** [comment]
LGTM!

**Comment by leoadb** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by leoadb** [comment]
Sent to https://github.com/liferay/com-liferay-dynamic-data-mapping/pull/623

**Pull request merged**